app/Http/Controllers/StaticContentController.php

```diff
@@ -0,0 +1,42 @@
+<?php
+
+namespace App\Http\Controllers;
+
+class StaticContentController extends Controller
+{
+    public function welcome() {
+        return view('welcome');
+    }
+
+    public function profile() {
+        return view('profile');
+    }
+
+    public function dashboard() {
+        return view('dashboard');
+    }
+
+    public function faq() {
```

Comment:
You have a FAQController. It should be responsible for handling all FAQ related requests. It is better to move this method there.